addOns/pscanrules/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [36] - 2021-08-13
+### Fixed
+- Reduce false positive from passive scan for private ip disclosure
```

Comment:
```suggestion
### Fixed
- Reduce potential false positives from passive scan rule for private IP disclosure
```

As noted earlier it should be in the unreleased section. The version, date, link are all updated as part of the release process.